Source/Core/Pipeline/010_AssignToStackAction.cs

```diff
@@ -58,7 +58,8 @@ public class AssignToStackAction : EventPipelineActionBase {
                         Logger.Trace().Message("Creating new event stack.").Write();
                         ctx.IsNew = true;
 
-                        string title = _formattingPluginManager.GetStackTitle(ctx.Event);
+                        var msi = ctx.Event.GetManualStackingInfo();
+                        string title = !String.IsNullOrWhiteSpace(msi?.Title) ? msi.Title : _formattingPluginManager.GetStackTitle(ctx.Event);
```

Comment:
Was wondering if I should make this into a stack formatting plugin... But I think this is fine.